tests/sync/test_git.py

```diff
@@ -121,6 +121,16 @@ def test_repo_git_obtain_full(
                 "dir": tmp_path / "myrepo",
                 "vcs": "git",
             },
+            False,
+        ],
+        [
+            GitSync,
+            lambda git_remote_repo, tmp_path, **kwargs: {
```

Comment:
You made sense of  the lazily interpreted fixtures! Well done!

(you can resolve this)